addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1359,7 +1359,9 @@ const Wysiwyg = Widget.extend({
             if (wasImageOrVideo || wasFontAwesome) {
                 $node.replaceWith(element);
                 this.odooEditor.unbreakableStepUnactive();
-                this.odooEditor.historyStep();
+                if (!(wasFontAwesome && element.classList.contains('fa'))) {
+                    this.odooEditor.historyStep();
```

Comment:
I don't understand. The extra step in the observer was only made when switching icons, but not for the other cases?

Reply:
Yes , I'm not sure why though

Reply:
> Yes , I'm not sure why though

Well, then I'm kind of obligated to say that this is fixing the symptom and not the root cause since the root cause would require finding what is making extra steps with the fa icons... ^^'